transport.go

```diff
@@ -134,3 +135,19 @@ func (t *Transport) refreshToken() error {
 
 	return nil
 }
+
+func addAcceptHeader(headers http.Header) {
+	if headers.Get("Accept") == "" {
+		headers.Set("Accept", acceptHeader)
+		return
+	}
+
+	//Need to loop through all Accept headers incase there is more than one.
+	for _, header := range headers["Accept"] {
+		//Looks as though all media types (https://developer.github.com/v3/media/) that can accept json end with "json". Only doing a suffix check to see if a json header already exists.
```

Comment:
Here as well:

```
// Looks as though ...
```

You might also want to wrap the line at around 80-100 columns, so it’s more comfortable to read.